controllers/topics.controller.js

```diff
@@ -0,0 +1,17 @@
+const { fetchTopics, fetchAPI } = require("../models/topics.models.js");
+const jsonData = require("../endpoints.json");
+
+function getTopics(req, res, next) {
+  return fetchTopics()
+    .then((topics) => res.status(200).send({ topics }))
+    .catch(next);
+}
+
+function getAPI(req, res, next) {
+  return Promise.all([fetchAPI()]).then((data) => {
```

Comment:
Not sure why this is Promise.all, also if you're not touching the database (you're not) you shouldn't need a model, this endpoint can be achieved within the controller 👍